File: BackEnd-Siukat/models/value.go
package models

type Value struct {
	IDValue   int    `gorm:"primaryKey;column:id_value" json:"id_value"`
	NoPeserta int    `gorm:"column:no_peserta" json:"no_peserta"`
	V1        int    `gorm:"column:v1" json:"v1"`
	V2        int    `gorm:"column:v2" json:"v2"`
	V3        int    `gorm:"column:v3" json:"v3"`
	V4        int    `gorm:"column:v4" json:"v4"`
	V5        int    `gorm:"column:v5" json:"v5"`
	Av1       int    `gorm:"column:av1" json:"av1"`
	Bv2       int    `gorm:"column:bv2" json:"bv2"`
	Cv3       int    `gorm:"column:cv3" json:"cv3"`
	Dv4       int    `gorm:"column:dv4" json:"dv4"`
	Ev5       int    `gorm:"column:ev5" json:"ev5"`
	Ikb       int    `gorm:"column:ikb" json:"ikb"`
	Atribut   string `gorm:"column:atribut;type:enum('original','sanggah')" json:"atribut"`
}

func (Value) TableName() string {
	return "tb_value"
}

File: ZhammDataBeLama/siukat-service-dev_snmptn_2022/models/value.js
var Sequelize = require('sequelize')
var db = require('../config/database')

const Value = db.define('value', {
  id_value:{
  	type: Sequelize.INTEGER,
  	primaryKey: true
  },
  no_peserta: {
    type: Sequelize.INTEGER
  },
  v1: {
    type: Sequelize.INTEGER
  },
  v2: {
    type: Sequelize.INTEGER
  },
  v3: {
    type: Sequelize.INTEGER
  },
  v4: {
    type: Sequelize.INTEGER
  },
  v5: {
    type: Sequelize.INTEGER
  },
  av1: {
    type: Sequelize.INTEGER
  },
  bv2: {
    type: Sequelize.INTEGER
  },
  cv3: {
    type: Sequelize.INTEGER
  },
  dv4: {
    type: Sequelize.INTEGER
  },
  ev5: {
    type: Sequelize.INTEGER
  },
  ikb: {
    type: Sequelize.INTEGER
  },
  atribut: {
    type: Sequelize.ENUM('original', 'sanggah')
  },
},
 {
  // don't add the timestamp attributes (updatedAt, createdAt)
  timestamps: false,

  // don't delete database entries but set the newly added attribute deletedAt
  // to the current date (when deletion was done). paranoid will only work if
  // timestamps are enabled
  paranoid: true,

  // don't use camelcase for automatically added attributes but underscore style
  // so updatedAt will be updated_at
  underscored: true,

  // disable the modification of tablenames; By default, sequelize will automatically
  // transform all passed model names (first parameter of define) into plural.
  // if you don't want that, set the following
  freezeTableName: true,

  // define the table's name
  tableName: 'tb_value'
}
);
module.exports = Value
